src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js

```diff
@@ -7,29 +7,53 @@ import Events from 'parser/core/Events';
 import SpellIcon from 'common/SpellIcon';
 import ItemHealingDone from 'interface/ItemHealingDone';
 import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import { TooltipElement } from 'common/Tooltip';
+import { formatNumber } from 'common/format';
 
 class InvokeChiJi extends Analyzer {
-  healing = 0;
+  gustHealing = 0;
+  envbHealing = 0;
 
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id);
     if (!this.active) return;
     this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.GUST_OF_MISTS_CHIJI), this.handleGust);
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.ENVELOPING_BREATH), this.handleEnvelopingBreath);
   }
 
   handleGust(event) {
-    this.healing += (event.amount || 0) + (event.absorbed || 0);
+    this.gustHealing += (event.amount || 0) + (event.absorbed || 0);
+  }
+
+  handleEnvelopingBreath(event) {
+    this.envbHealing += (event.amount || 0) + (event.absorbed || 0);
   }
 
   statistic() {
     return (
-      <StatisticBox
-        position={STATISTIC_ORDER.OPTIONAL(50)}
-        icon={<SpellIcon id={SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id} />}
-        value={<ItemHealingDone amount={this.healing} />}
-        label="Chi-Ji Healing"
-      />
+      <>
+        <StatisticBox
```

Comment:
We have deprecated StatisticBox and moved to Statistic as our new layout for these things!

Reply:
![image](https://user-images.githubusercontent.com/26779541/94999043-76176a00-0584-11eb-9743-63eef003859c.png)
updated